src/components/Chart/ChartSquare.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import classNameContext from 'classnames';
+
+export const ChartSquare = ({ rating, seasonNumber, episodeNumber, type }) => {
+  // TODO: create a more verbose hex decimal color scheme
+  const getColor = (episodeRating) => {
+    let color = 'white';
+    if (episodeRating <= 4) {
+      color = 'red';
+    } else if (episodeRating <= 7) {
+      color = 'yellow';
+    } else {
+      color = 'green';
+    }
+    return color;
+  };
+
+  const style = {
+    gridColumnStart: episodeNumber,
+    gridColumnEnd: episodeNumber + 1,
+    gridRowStart: seasonNumber,
+    gridRowEnd: seasonNumber + 1,
+    backgroundColor: type === 'episodeData' && getColor(rating),
+  };
+
+  const chartSquareClassNames = classNameContext({
+    'chart-square': type === 'episodeData',
+    'season-labels': type === 'seasonLabels',
+    'episode-labels': type === 'episodeLabels',
+  });
+
+  const text = (input) => {
```

Comment:
might want to rename this, not sure what yet

Reply:
resolved